features/user_sessions.feature

```diff
@@ -46,3 +46,66 @@ Feature: User sessions
     When I sign out
     And I go to the usagov's Dashboard page
     Then I should see "Security Notification"
+  
+  @javascript
+  Scenario: User sees the header with USWDS banner
+    When I go to the login page
+    Then I should see "An official website of the United States government"
```

Comment:
Please add `And I should not see "Official websites use .gov"` before the button press; this confirms that the partial is hidden until the user clicks the button.

Reply:
Sure, Sounds good.